### PR TITLE
refactor: trim package

### LINKED ISSUE #1: Organize repository
Organize this repo.

- Update README
- Remove unrelated codes (Including this GitHub issue templates, etc
- Use GB's eslint
- Publish package under `@globalbrain` namespace.
- etc.

### PR DESCRIPTION
**Pull request by cuebit**
resolves #1

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -20,8 +20,14 @@ export function getRouteName(routeName?: string | symbol | null) {
...
+): string {
+  return getLocalizeRouteName(getRouteName(routeName), locale)
+}
+
+export function getLocalizeRouteName(
+  routeName: string | null,
+  locale: string
+): string {
+  return `${routeName}___${locale}`
 }
```
Comment (cuebit) [inline src/utils.ts:33]: Added to avoid hardcoding in multiple places.

Comment (cuebit) [inline src/utils.ts]: Added to avoid hardcoding in multiple places.

**Review by brc-dd — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,17 +1,16 @@
 {
-  "name": "@brc-dd/nuxt-i18n",
+  "name": "@globalbrain/nuxt-i18n",
   "type": "module",
   "version": "1.0.20",
-  "packageManager": "pnpm@8.10.5",
+  "packageManager": "pnpm@8.15.4",
   "description": "Minimal module for i18n with locale auto-imports & localized routing",
-  "author": "Johann Schopplich <pkg@johannschopplich.com>",
+  "author": "Kia Ishii <ka.ishii@globalbrains.com>",
```
Comment (kiaking) [inline package.json]: I think it should be: 

```suggestion
  "author": "Divyansh Singh <divyansh@globalbrains.com>",
```

@brc-dd ☝️ Please check this credit is correct (like email address

**Review by kiaking — APPROVED** [review]
Made tiny adjustment on README to align it with eslint config.

@brc-dd 
Made only one comment. Once this is solved, let's merge and release 🙌

File: package.json (hunk 2 above)
Comment (brc-dd) [reply package.json]: Yeah, lol 😅

**Pull request merged**